tests/resource_aws_organizations_policy/.terraform-version

```diff
@@ -0,0 +1 @@
+0.13.4
```

Comment:
This new testing is nice because its covering this very specific scenario, but I have a few concerns:

* This testing is outside of currently automated test runs (when would this get run again? when should it?)
* This new directory structure and testing styles is not self-documented or in the contributing/maintaining guide
* Introduces a new maintenance burden of maintaining a Terraform version here (is the version number significant? should it instead be in the configuration as `required_version = ">= 0.12"`?)

I think we should probably work through some of these before introducing it into the codebase.

Reply:
For sure. I included it here essentially to "show my work" with testing the change. I expected it to be iterated on before becoming something official. I'll pull it out and make a WIP proposal